frontend/server/src/Controllers/Run.php

```diff
@@ -915,6 +921,144 @@ public static function apiDetails(\OmegaUp\Request $r): array {
             $response['judged_by'] = strval($run->judged_by);
         }
 
+        $problemArtifacts = new \OmegaUp\ProblemArtifacts(
+            $problem->alias,
+            $run->commit
+        );
+
+        $cases = self::getCases(
+            $problemArtifacts,
+            'cases',
+            $problem->alias,
+            $run->commit
+        );
+        /** @var int $responseSize */
+        $responseSize = array_reduce(
+            $cases,
+            /**
+             * @param array{mode: int, type: string, id: string, size: int, path: string} $case
+             */
+            function (?int $sum, $case): int {
+                if (is_null($sum)) {
+                    return $case['size'];
+                }
+                $sum += $case['size'];
+                return $sum;
+            }
+        );
```

Comment:
```suggestion
        $responseSize = array_reduce(
            $cases,
            /**
             * @param array{mode: int, type: string, id: string, size: int, path: string} $case
             */
            function (int $sum, $case): int {
                return $sum + $case['size'];
            },
            0
        );
```

si le pasas el `$initial` como 0, ya no hay necesidad de tratar el caso especial donde el argumento es nulo!

Reply:
Listo